Tests/AuthTests/CheckPasswordRecoveryGuidTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Moq;
+
+namespace Tests.AuthTests
+{
+    public class CheckPasswordRecoveryGuidTests : BaseAuthServiceTest
+    {
+        [Fact]
+        public async Task CheckPasswordRecoveryGuid_ShouldReturnTrue_WhenUserInDBAndGuidAreTheSame()
+        {
+            //Arrange
+            Guid inputGuid = Guid.NewGuid();
+            string inputEmail = "test@gmail.com";
+            Guid? userGuidInDb = inputGuid;
+
+            UserRepositoryMock.Setup(x => x.GetUserGuidByEmail(inputEmail)).Returns(userGuidInDb);
+
+            //Act
+            bool check = await sut.CheckPasswordRecoveryGuid(inputGuid, inputEmail);
```

Comment:
What is `sut` ? This name doesn't say anything

Reply:
system under test